courseplay.lua

```diff
@@ -173,18 +173,14 @@ local function setVersionData()
 	end;
 
 	if courseplay.version then
-		local versionSplitStr = StringUtil.splitString('.', courseplay.version); -- split as strings
-		versionSplitStr[3] = versionSplitStr[3] or '0000';
-		courseplay.versionDisplayStr = string.format('v%s.%s\n.%s', versionSplitStr[1], versionSplitStr[2], versionSplitStr[3]); --multiline display string
-		courseplay.isDevVersion = tonumber(versionSplitStr[3]) > 0;
+		courseplay.versionDisplayStr = string.format("v%s",courseplay.version)
+		courseplay.isDevVersion = true
```

Comment:
The idea here was I think to follow the semantic versioning scheme (major.minor.patch, see https://semver.org/), with the last number of the three being 0 is an official modhub release, incremented between releases as we fix issues, then the next modhub release would increase the minor version. It is a 'dev' release if the patch number is not 0.

With this change we'd always show 'dev', so I'd rather get rid of that 4th number of the version string and return to the major.minor.patch scheme.

Reply:
We must use 4 Numbers for ModHub, due to GIANTS Rules.
I like the old system more too, but GIANTS don't want that...

Reply:
Oh, I see, then either:
* change this code to check the 4th number instead of the 3rd to decide if 'dev' should be displayed, or
* just display the version string and nothing else, don't worry about adding 'dev' to it.